packages/store/src/patterns/patternMatchers.js

```diff
@@ -732,6 +803,105 @@ const makePatternKit = () => {
     },
   });
 
+  /** @type {MatchHelper} */
+  const matchBigintHelper = Far('match:bigint helper', {
+    checkMatches: (specimen, [limits = undefined], check) => {
+      const { decimalDigitsLimit } = limit(limits);
+      const absSpecimen = specimen < 0n ? -specimen : specimen;
+      return (
+        checkKind(specimen, 'bigint', check) &&
+        check(
+          `${absSpecimen}`.length <= decimalDigitsLimit,
```

Comment:
Good idea! But your formula seemed off by 2, and I don't see the purpose of the `ceil` since it's just feeding a floating point inequality. I'm going with
```js
          Math.log10(Math.abs(Number(specimen))) <= decimalDigitsLimit,
```

Reply:
Unresolving. I think your formula was more correct than mine.

Reply:
Done. your's was closer.
```js
Math.floor(Math.log10(Math.abs(Number(specimen)))) + 1 <= decimalDigitsLimit
```